packages/frontend/editor-ui/src/components/HtmlEditor/utils.ts

```diff
@@ -38,3 +40,25 @@ export function nonTakenRanges(fullRange: Range, takenRanges: Range[]) {
 
 	return found;
 }
+
+export function pasteHandler(event: ClipboardEvent, view: EditorView) {
+	const htmlData = event.clipboardData?.getData('text/html') ?? '';
+	const textData = event.clipboardData?.getData('text/plain') ?? '';
+
+	const content = htmlData || textData;
+	if (!content) return false;
+
+	// Replace onclick="" with onclick=" " as empty onclick breaks range parser
+	if (/onclick=""/.test(content)) {
+		event.preventDefault();
+		const sanitized = content.replace(/onclick=""/g, 'onclick=" "');
```

Comment:
Is this bug only for onclick? or any empty attribute?

Reply:
I did try several other, looks like only applies to onclick